apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -156,6 +156,13 @@ type VSHNPostgreSQLServiceSpec struct {
 	// +kubebuilder:default=false
 	DisablePgBouncer bool `json:"disablePgBouncer,omitempty"`
 
+	// Set to true to reserve the specified CPU and memory exclusively for Patroni,
+	// with sidecar containers getting separate allocations.
+	// Leave as false (default) to let StackGres automatically distribute the
+	// specified resources across all containers.
+	// +kubebuilder:default=false
+	DedicatedPatroniResources bool `json:"dedicatedPatroniResources,omitempty"`
```

Comment:
True should be the default. This makes the unsafe/unexpected case the default.

A stable database (for new instances and as default) should be the goal over bloating the resources.

Reply:
Then why are we even doing it? If we don't get QOS guaranteed from this change it's not necessary no?